### ISSUE TITLE
scan multiple locations

### ISSUE THREAD

**Issue by deletenhp** [engle2192/PoGoMap-GUI#24]
Is there a way I can use this to scan multiple locations? I have been pouring through all of the documentation for pokemongomap and it looks possible, I just don't see how I can do it with this. The area I want to scan is an island, and its about 4 miles long but only a few blocks wide with the widest point maybe a mile max. Trying to eliminate scanning in the water and slowing things down. I was running multiple instances of pokemongomap desktop, but this is much more reliable and I am enjoying using pokealarm. Any ideas? Can I just run multiple instances?

**Comment by Go2Engle** [comment]
I would just turn the scan distance up real high to cover the whole area you want. Then run the scan for a few hours. AFter you run for a few hours turn it off and check spawn point scanning.

**Comment by deletenhp** [comment]
Thank you. That is actually what I ended up doing. I had to use about 50 workers to scan 30 steps in about 7/8 minutes. Ran that overnight and then switched to spawn point scanning. My pokealerts usually come through with 14 minutes 30 seconds left, things are working marvelous. You really did a great job! I haven't written python since college, and this really streamlines the whole process so well.
